packages/api-security/src/plugins/models/securityUser.model.ts

```diff
@@ -41,16 +40,6 @@ export default ({ createBase, context }): any => {
                     validation: validation.create("required")
                 })
             ),
-            password: onSet(value => {
```

Comment:
OMG @Pavel910 can't believe this is still here :D

Reply:
@doitadrian Yeah, thought it's time to retire that 😆